docs/static/releasenotes.asciidoc

```diff
@@ -24,6 +26,97 @@ This section summarizes the changes in the following releases:
 * <<logstash-6-1-1,Logstash 6.1.1>>
 * <<logstash-6-1-0,Logstash 6.1.0>>
 
+[[logstash-6-6-0]]
+=== Logstash 6.6.0 Release Notes
+
+
+=== Logstash Pull Requests with label v6.6.0
+
+* BUGFIX: Remove excess period from logstash-plugin error reporting https://github.com/elastic/logstash/pull/9749[#9749]
+* Enhancement: Replace generated for-loops with static method calls https://github.com/elastic/logstash/pull/10133[#10133]
+* Enhancement: Add common base class for generated Dataset classes https://github.com/elastic/logstash/pull/10136[#10136]
+* Upgrade rack to version 1.6.11  https://github.com/elastic/logstash/pull/10151[#10151]
+* Enhancement: Java plugin API (experimental) https://github.com/elastic/logstash/pull/10216[#10216]
+* BUGFIX: Increase default number of threads reported by hot_threads https://github.com/elastic/logstash/pull/10218[#10218]
+* BUGFIX: Add option for PQ checkpoint retry https://github.com/elastic/logstash/pull/10246[#10246]
+* Add documentation for java-execution flag https://github.com/elastic/logstash/pull/10248[#10248]
+* BUGFIX: Properly set thread name in thread context https://github.com/elastic/logstash/pull/10272[#10272]
+* Announcement: HTTP lookup filter and Memcached get/set filter now GA.
+
+==== Plugins
+
+*Aggregate Filter*
+
+* new feature: add ability to dynamically define a custom `timeout` or `inactivity_timeout` in `code` block (fix issues [#91](https://github.com/logstash-plugins/logstash-filter-aggregate/issues/91) and [#92](https://github.com/logstash-plugins/logstash-filter-aggregate/issues/92))
```

Comment:
The MD link format doesn't render correctly in asciidoc. Should be:
https://github.com/logstash-plugins/logstash-filter-aggregate/issues/91[#91]

The links in the Logstash section are correct.  Only the plugin links are affected. (I didn't flag them all, but can if that would be helpful.)